stix2/constants.py

```diff
@@ -0,0 +1,135 @@
+import base64
+import binascii
+import re
+
+# TODO: REConstant?
+# TODO: Timestamp
+
+
+class Constant(object):
+    def __str__(self):
+        return "%s" % self.value
+
+    @staticmethod
+    def escape_quotes_and_backslashes(s):
```

Comment:
Ha, I get into those moods occasionally. Now I'm in a "make everything a module-level function" mood.